scripts/compsets/config/wcoss.phase2.config

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
We've dropped Phase 2 support; this and the symlinked phase 1 can be deleted.

Reply:
Agreed. Please proceed with the change.